Automattic/newspack-blocks
# Add 2 new subscribe patterns

### All Submissions:






### Changes proposed in this Pull Request:

This PR adds 2 new subscribe patterns, including one without any image.

These 2 patterns are based on Hechinger Report (except here I'm using Jetpack Mailchimp). E.g. https://hechingerreport.org/desperate-parents-need-help-as-coronavirus-upends-our-lives/

![Screenshot 2020-03-31 at 18 50 29](https://user-images.githubusercontent.com/177929/78058739-a9343300-7380-11ea-8a99-96f734e76f90.png)

### How to test the changes in this Pull Request:

1. Switch to this branch
2. Edit a page/post/popup. Do you see the new patterns?
3. Try to add them to your page/post/popup.

### Other information:
